css/custom.css

```diff
@@ -637,4 +637,7 @@ modal-body {
   margin:0 auto;
   margin-top:10px;
   margin-bottom:10px;
-}
\ No newline at end of file
+}
+.pull-right-glyphicon {
+  float:right;
+}
```

Comment:
Pretty disappointing to be back to this. Perhaps we're speaking past each other. Bootstrap already has a helper class to do this - https://getbootstrap.com/docs/3.3/css/#helper-classes-floats

Reply:
@DavidLemayian I am sorry about this. I didn't understand  a bootstrap class was already taking care of it.